saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -119,9 +123,20 @@ public String getAssertionConsumerServiceUrlTemplate() {
 	 * Contains the URL for which to send the SAML 2 Authentication Request to initiate
 	 * a single sign on flow.
 	 * @return a IDP URL that accepts REDIRECT or POST binding for authentication requests
+	 * @deprecated use {@link ProviderDetails#getWebSsoUrl()} from {@link #getProviderDetails()}
 	 */
+	@Deprecated
```

Comment:
Since this is being deprecated, please remove any references to it in the code.

Reply:
Completed.
c7937d150082ac8fd4a1ede9556876e10985d820